ibis/backends/tests/test_struct.py

```diff
@@ -29,6 +31,14 @@
 ]
 
 
+@pytest.mark.notimpl("duckdb", raises=DuckDBParserException)
+@pytest.mark.notimpl("polars", raises=PolarsComputeError)
+def test_struct_factory_empty(con):
+    s = ibis.struct({})
+    result = con.execute(s)
+    assert result == {}
```

Comment:
I don't think this is supported in many of our backends. How is this spelled in DuckDB for example?

Reply:
moving to a top-level thread in this PR.